packages/create-gatsby/src/index.ts

```diff
@@ -79,6 +82,7 @@ export const questions = (initialFolderName: string): any => [
     initial: initialFolderName,
     format: (value: string): string => c.cyan(value),
     validate: validateProjectName,
+    skip,
```

Comment:
Is this still needed?

Reply:
Yes, we're still calling this question with an initial folder name to process the results but skipping it so that the user isn't prompted. Theoretically we could create the data object directly, but this seemed cleaner.